tests/dedupe.js

```diff
@@ -31,7 +31,7 @@ describe('dedupe', function () {
 	});
 
 	it('joins arrays of class names and ignore falsy values', function () {
-		assert.equal(dedupe('a', 0, null, undefined, true, 1, 'b'), '1 a b');
```

Comment:
I don't know if this is actually deterministic

Reply:
When you iterate through a Set you receive the values in insertion order, so this should be deterministic.